libs/src/api/Track.ts

```diff
@@ -512,7 +535,10 @@ export class Track extends Base {
    * Adds tracks to chain for this user
    * Associates tracks with user on creatorNode
    */
-  async addTracksToChainAndCnode(trackMultihashAndUUIDList: ChainInfo[]) {
+  async addTracksToChainAndCnode(
+    trackMultihashAndUUIDList: ChainInfo[],
+    useEntityManager: boolean
```

Comment:
would it make sense adding `useEntityManager` as a field on the Track api to internally reference on these methods? or do we want to specify this boolean for each method call?